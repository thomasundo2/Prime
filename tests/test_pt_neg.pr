int main()
{
    /* creation and assignment */
    pt p1;
    pt p2;
    pt p3;
    poly crv;

    crv = [(5l, 12l) : 13l];
    p1 = [2l, 2l] & crv;
    p2 = -p1;
    printpt(-p1);
    printpt(p2);
    printpt(-[2l, 2l] & crv);
    printpt(-[-1l, -1l] & crv);
    return 0;
}
