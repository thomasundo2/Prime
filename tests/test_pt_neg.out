[2,11] & [(5,12) : 13]
[2,11] & [(5,12) : 13]
[2,11] & [(5,12) : 13]
[-1,-1] & [(5,12) : 13]
